### ISSUE TITLE
Cyclomatic complexity is 2 if class has only one method

### ISSUE THREAD

**Issue by Juanlu001** [rubik/radon#70]
Minimal working example:

```shell
$ python -c "import radon; print radon.__version__"
1.1.1

$ cat tests/test_dummy.py
class Foo0:
    pass

class Foo1:
    def test1(self): pass

class Foo2:
    def test1(self): pass
    def test2(self): pass

$ radon cc -as tests/test_dummy.py
tests/test_dummy.py
    C 4:0 Foo1 - A (2)
    C 1:0 Foo0 - A (1)
    M 5:4 Foo1.test1 - A (1)
    C 7:0 Foo2 - A (1)
    M 8:4 Foo2.test1 - A (1)
    M 9:4 Foo2.test2 - A (1)

6 blocks (classes, functions, methods
Average complexity: A (1.16666666667)
```

**Comment by rubik** [comment]
I've modified the code a bit. Do you think this result makes sense?
```
 radon cc /tmp/test.py -as
/tmp/test.py
    C 1:0 Foo0 - A (1)
    C 4:0 Foo1 - A (1)
    M 5:4 Foo1.test1 - A (1)
    C 7:0 Foo2 - A (1)
    M 8:4 Foo2.test1 - A (1)
    M 9:4 Foo2.test2 - A (1)
```

**Comment by Juanlu001** [comment]
Cool, just as I expected! From my point of view this can be closed. Are you planning a release soon?

**Comment by rubik** [comment]
Yes, I will release now.

**Comment by rubik** [comment]
I released radon 1.2 to PyPI.
